test/tests.js

```diff
@@ -451,6 +451,19 @@ QUnit.test('Call to read all cookies with mixed json', function (assert) {
 	assert.deepEqual(Cookies.get(), { c: '{"foo":"bar"}', c2: 'v' }, 'returns unparsed cookies');
 });
 
+QUnit.test('Cookies with escaped quotes in json', function (assert) {
+	Cookies.withConverter({
+		read: function (value) {
+			return value;
+		},
+		write: function (value) {
+			return value;
+		}
+	}).set('c', '"{ \\"foo\\": \\"bar\\" }"');
+	assert.deepEqual(Cookies.getJSON('c'), '{ "foo": "bar" }', 'returns JSON parsed cookie');
```

Comment:
We don't need `deepEqual` here. Deep quality is when we need to compare the structure of Literals such as `[...]` or `{...}`. We don't need that for String which is an immutable data structure. Can we use `assert.strictEqual` instead, so it compares using `===`?